packages/core-js/scripts/postinstall.js

```diff
@@ -15,10 +24,12 @@ function log(it) {
   console.log(COLOR ? it : it.replace(/\u001B\[\d+m/g, ''));
```

Comment:
Could `console.info` be used instead of `console.log`? `console.log` is not properly silenced when the `--loglevel=warn` argument is used.